.circleci/config.yml

```diff
@@ -1,100 +0,0 @@
-version: 2
-jobs:
-  build:
-    environment:
-      - SBT: "sbt -v -mem 2048"
-    docker:
-      - image: rtkaczyk/mantis-circleci:v6
-    steps:
-      - checkout
-
-      - restore_cache:
-          key: mantis-{{ checksum "build.sbt" }}-{{ checksum "project/build.properties" }}-{{ checksum "project/plugins.sbt" }}-v1
-
-      - run:
-          name: submodule
-          command: |
-            git submodule init
-            git submodule update
-
-      - run:
-          name: scalastyle
-          command: $SBT scalastyle test:scalastyle
-
-      - run:
-          name: unit tests
-          command: $SBT coverage test
-
-      - run:
-          name: EVM tests
-          command: $SBT coverage evm:test
-
-      - run:
-          name: integration tests
-          command: $SBT coverageOff it:test
-
-      - run:
-          name: coverage report
-          command: $SBT coverageReport coverageAggregate
-
-      - store_artifacts:
-          path: target/scala-2.12/coverage-report
-          destination: scala-2.12/coverage-report
-
-      - store_artifacts:
-          path: target/scala-2.12/scoverage-report
-          destination: scala-2.12/scoverage-report
-
-      - run:
-          name: ETS
-          command: ./test-ets.sh
-          no_output_timeout: 1h
-
-      - run:
-          name: additional compilation & dist
-          # this step builds parts of the codebase which are not tested in CI
-          # so as to prevent compilation regression
-          command: $SBT benchmark:compile snappy:compile dist
-
-      - save_cache:
-          key: mantis-{{ checksum "build.sbt" }}-{{ checksum "project/build.properties" }}-{{ checksum "project/plugins.sbt" }}-v1
-          paths:
-            - ~/.sbt
-            - ~/.ivy2/cache
-            - ~/.m2
-
-
-  full_ETS:
-    docker:
-      - image: rtkaczyk/mantis-circleci:v6
-    steps:
-      - checkout
-
-      - restore_cache:
-          key: mantis-{{ checksum "build.sbt" }}-{{ checksum "project/build.properties" }}-{{ checksum "project/plugins.sbt" }}-v1
-
-      - run:
-          name: FULL ETS
-          command: RUN_FULL_ETS=yes ./test-ets.sh
```

Comment:
Not for the scope of this task, but we should think how we could add this into our CI flow. Current main issue is that it take to much time.